src/lib/constants.ts

```diff
@@ -31,3 +31,14 @@ export const HTTP_METHODS: HttpMethod[] = [
 ] as const;
 
 export const DEFAULT_HTTP_METHOD: HttpMethod = 'GET';
+
```

Comment:
вот такой тип в types:

```suggestion
export type Language =
  | 'cURL'
  | 'JavaScript Fetch'
  | 'JavaScript XHR'
  | 'NodeJS'
  | 'Python'
  | 'Java'
  | 'C#'
  | 'Go';
```